src/sentry/static/sentry/app/views/eventsV2/table/cellAction.tsx

```diff
@@ -113,16 +133,64 @@ export default class CellAction extends React.Component<Props, State> {
         }
         query[negation].push(`${value}`);
         break;
+      case Actions.SHOW_GREATER_THAN: {
+        // Remove query token if it already exists
+        delete query[`${column.name}`];
```

Comment:
Why do you need a string cast here and not on the following line?

Reply:
done in https://github.com/getsentry/sentry/pull/19230/commits/9b04e0ed124fa7185337eb4b89ed1266ac819605